docs/platform/howto/use-google-private-service-connect.rst

```diff
@@ -0,0 +1,175 @@
+
+Select “Published service” as “Target type”. “Target service” should be the GOOGLE_SERVICE_ATTACHMENT value/URI from the previous step.
+
+Select an existing subnet hosting your side of the endpoint.
+
+After the endpoint has been created, it will initially exist in a "pending" state. To allow connections via the endpoint, it needs to be accepted at the service publisher (Aiven) end.
+
+If you used automatically assigned IP address, note the IP address associated with the endpoint for the next step.
+
+Step 3: Approve the created connection for an Aiven service
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, update the state of Private Service Connect connections for you Aiven service by running
+
+.. code:: shell
+
+    avn service privatelink google refresh MY_SERVICE_NAME
+
+Wait until the following command (which you can retry until it does), returns pending-user-approval status:
+
+.. code:: shell
+
+    avn service privatelink google connection list MY_SERVICE_NAME
+
+.. code:: shell
+
+    PRIVATELINK_CONNECTION_ID  PSC_CONNECTION_ID  STATE                  USER_IP_ADDRESS
+    =========================  =================  =====================  ===============
+    plc3fd852bec98             12870921937223780  pending-user-approval  null
+
+PSC_CONNECTION_ID is the identifier assigned to Google for the connection, and you can use it to verify that the connection is indeed matching your PSC endpoint.
+
+PRIVATELINK_CONNECTION_ID is Aiven internal identifier for the connection, you'll need in the final connection approval step below.
+
+To enable a connection, you should approve it, at the same time providing the IP address assigned to your PSC endpoint - whether automatically assigned or static. Aiven will use this IP address for pointing the service DNS records necessary for the clients to reach the Aiven service through the PSC connection.
+
+To complete the connection, you need to issue approval command as follows:
+
+.. code:: shell
+
+    avn privatelink google connection approve MY_SERVICE_NAME --privatelink-connection-id PRIVATELINK_CONNECTION_ID --user-ip-address PSC_ENDPOINT_IP_ADDRESS
+
+The connection will initially transition to user-approved state:
+
+.. code:: shell
+
+    avn service privatelink google connection list MY_SERVICE_NAME
+
+.. code:: shell
+
+    PRIVATELINK_CONNECTION_ID  PSC_CONNECTION_ID  STATE          USER_IP_ADDRESS
+    =========================  =================  =============  ===============
+    plc3fd852bec98             12870921937223780  user-approved  10.0.0.100
+
+An extra
+
+.. code:: shell
+
+    avn service privatelink google refresh MY_SERVICE_NAME
+
+may be needed at this point, because updates to service attachment accept lists are not immediately reflected in the states of returned connected endpoints.
```

Comment:
```
You may be need to run the ``avn service privatelink google refresh`` command at this point since updates to service attachment accept lists are not immediately reflected in the states of returned connected endpoints.

.. code:: shell
    avn service privatelink google refresh MY_SERVICE_NAME
```